### ISSUE TITLE
Plugin icons visible by default

### ISSUE THREAD

**Issue by lonesomewalker** [chamilo/chamilo-lms#3246]
**Describe**
If you activate a plugin and this is an activity like BigBlueButton, the icon of this becomes visible by default.
There is no option do say: this plugin/item is also deactivated upon course creation.

**Comment by ywarnier** [comment]
I agree this is annoying. When disabling the plugin (with the plugin_enable setting inside the plugin), the plugin icons should be removed from the course homepage automatically.

**Comment by ywarnier** [comment]
To do:
- when enabling a course plugin, the icon is added to all courses' homepages with the visibility defined inside the plugin by variable "defaultVisibilityInCourseHomepage"
- when disabling a course plugin, the icon is *removed* from all courses' homepages
- when a *new* course is created, the course homepage should include the plugin icon with the default visibility configured in the plugin

Add the defaultVisibilityInCourseHomepage setting in the default plugin class, then check if it is a course plugin before showing the option in the plugin settings page

**Comment by christianbeeznest** [comment]
It is implemented with this PR https://github.com/chamilo/chamilo-lms/pull/6626

Thanks for confirmation.

**Comment by ywarnier** [comment]
This works well. There's still work to do on the plugins "enabling", as described in issue #6649
